### ISSUE TITLE
SR-specific contact page changes

### ISSUE THREAD

**Issue by jlyon** [proudcity/wp-proudcity#510]
Extension to #507 

- [x] SR only: Change Email on Department list label to something else "Contact" #514
- [x] SR only: Dept list: Remove person col #511 
- [x] SR only: Dept and People list: Hide social field #511

**Comment by jlyon** [comment]
@kevindherman I added new pills style to the Submenu widget, and then reorganized these pages:
1. Created Contact menu
2. Replaced "Contact submenu" widget with Submenu widget using Contact menu and Pills style
3. Created dedicated pages with specific People lists for Elected Officials (https://san-rafael-ca.proudcity.com/people-elected/) and City Staff (https://san-rafael-ca.proudcity.com/people-city-staff/)
4. Created new /departments page with Agency list
5. Replaced /contact with a Romulus form (currently just the City Council feedback form)

Still todo
- [ ] Pill active state isn't correct
